src/App.tsx

```diff
@@ -7,55 +7,40 @@ import PizZipUtils from 'pizzip/utils/index.js';
 import { saveAs } from 'file-saver';
 import { Step1 } from './Steps/Step1';
 import { FinalStep } from './Steps/FinalStep';
-import AppHeader from './components/AppHeader'
-
-
+import AppHeader from './components/AppHeader';
+import { globalContext } from './stateManagement/GlobalStore';
+import { AlertModal } from './Steps/AlertModal';
 
 function App() {
-  const date = new Date();
+  const { globalState, dispatch } = useContext(globalContext);
   const [isLoading, setLoading] = useState(false);
-  const [options, setOptions] = useState<any>({
-    program_mod_system_name: "",
-    TMCRDate: date.toDateString(),
-    type_of_contract: "Firm Fixed Price",
-    attachment_number: null,
-    cdrl_sequence_number: null,
-    exhibit: null,
-    rfp_contract: null,
-    CLIN: null,
-    include_stuff: false
```

Comment:
@RobertPorterfield  -- Assuming the elements that went away and did not get included in the Global Wizard state will be created as they are needed based on Sprints, and that these were just initially here for testing?

Reply:
You got it! This was placeholder/demo stuff